src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -229,14 +229,39 @@ codeunit 396 NoSeriesManagement
         IsHandled: Boolean;
     begin
         IsHandled := false;
+#if not CLEAN24
         OnBeforeGetNextNo(NoSeriesCode, SeriesDate, ModifySeries, Result, IsHandled, LastNoSeriesLine);
+#endif
         if IsHandled then
             exit(Result);
 
         exit(DoGetNextNo(NoSeriesCode, SeriesDate, ModifySeries, false));
     end;
 
 #if not CLEAN24
+    /// <summary>
```

Comment:
Empty summary?

Reply:
I just wanted to have the remarks bit ... the rest was auto generated... it does look a bit odd now... and we don't usually doc internal methods... :-S